src/scripts/transformRecordedTests/upgradeThatMarks.ts

```diff
@@ -0,0 +1,48 @@
+import { TestCaseFixture } from "../../testUtil/TestCase";
```

Comment:
This file will be reverted.  It transforms all `that` marks to be the `UntypedTarget` that they would have become via `ThatMarkStage` before.